Mage/Task/BuiltIn/Deployment/Strategy/RsyncTask.php

```diff
@@ -100,7 +100,9 @@ public function run()
                  . $this->excludes($excludes) . ' '
                  . $this->excludesListFile($excludesListFilePath) . ' '
                  . $this->getConfig()->deployment('from') . ' '
-                 . $this->getConfig()->deployment('user') . '@' . $this->getConfig()->getHostName() . ':' . $deployToDirectory;
+                 . ( $this->getConfig()->deployment('user') ? $this->getConfig()->deployment('user') . '@' : '' )
```

Comment:
Hey, 

I've  seen that interesting stuff  recently, you would definitely  love it. Just take a look <http://between.hatterasyacht4sale.com/e6krtqz>

In haste, esmiz